src/Prismic/Api.php

```diff
@@ -161,58 +198,42 @@ public function master()
      * Returns all forms of type Prismic::SearchForm that are available for this repository's API.
      * The intended syntax of a call is: api->forms()->everything->query(query)->ref(ref)->submit().
      * Learn more about those keywords in prismic.io's documentation on our developers' portal.
-     *
-     * @return all forms
      */
-    public function forms()
+    public function forms() : stdClass
```

Comment:
shouldn't we have a interface Form?

Reply:
`forms()` returns an object where each key corresponds to a `Prismic\SearchForm` instance, so if anything, it should be a `SearchFormCollection` or something. Or, the method should be removed and replaced with something like `public function searchForm(string $name) : SearchForm` and be called `$api->searchForm('everything')->query(whatever)`

Reply:
we could just specify the type, Array[SearchForm], there is a way to declare this type as return type in php 7?

Reply:
Can't we specify a more explicit type like Array[SearchForm] in PHP?

Reply:
No, you can't do that. We could have a return type of `array` and a doc block `@return SearchForm[]` - the doc block will satisfy most IDEs. The alternative is to kill off the method and prefer something like I mentioned: `public function searchForm(string $name = 'everything') : SearchForm` - forms() probably should have always been private anyhow in favour of a more explicit method. All of this is a pretty significant BC break because most of the docs say to `$api->forms()->everything->query()` - If forms returned an object with public properties (Each a search form) then meaningful exceptions could be thrown for unknown form names, but this all falls apart if you name your form 'my-form' - this is valid in JSON, but in PHP you'd need $api->forms()->{'my-form'}->query()`